pig/src/main/java/org/apache/iceberg/pig/SchemaUtil.java

```diff
@@ -72,20 +72,31 @@ private static ResourceFieldSchema convert(Type type) throws IOException {
 
   private static byte convertType(Type type) throws IOException {
     switch (type.typeId()) {
-      case BOOLEAN:   return DataType.BOOLEAN;
-      case INTEGER:   return DataType.INTEGER;
-      case LONG:      return DataType.LONG;
-      case FLOAT:     return DataType.FLOAT;
-      case DOUBLE:    return DataType.DOUBLE;
-      case TIMESTAMP: return DataType.CHARARRAY;
-      case DATE:      return DataType.CHARARRAY;
-      case STRING:    return DataType.CHARARRAY;
-      case FIXED:     return DataType.BYTEARRAY;
-      case BINARY:    return DataType.BYTEARRAY;
-      case DECIMAL:   return DataType.BIGDECIMAL;
-      case STRUCT:    return DataType.TUPLE;
-      case LIST:      return DataType.BAG;
-      case MAP:       return DataType.MAP;
+      case BOOLEAN:
+        return DataType.BOOLEAN;
+      case INTEGER:
+        return DataType.INTEGER;
+      case LONG:
+        return DataType.LONG;
+      case FLOAT:
+        return DataType.FLOAT;
+      case DOUBLE:
+        return DataType.DOUBLE;
+      case TIMESTAMP:
+      case DATE:
+      case STRING:
+        return DataType.CHARARRAY;
+      case FIXED:
+      case BINARY:
+        return DataType.BYTEARRAY;
+      case DECIMAL:
+        return DataType.BIGDECIMAL;
+      case STRUCT:
+        return DataType.TUPLE;
+      case LIST:
+        return DataType.BAG;
+      case MAP:
+        return DataType.MAP;
```

Comment:
Unless there was a functional change, can you revert this? There's no need to change the formatting here.

Reply:
reverted